src/components/Navbar/Navbar.scss

```diff
@@ -1,18 +1,18 @@
-@import './../../styles/fonts.scss';
+// @import './../../styles/fonts.scss';
 
-.nav-text {
-  font-family: $poppins;
-  font-weight: 300;
-  font-size: 17px;
-  line-height: 25px;
-  text-shadow: 0px 2px 2px rgba(0, 0, 0, 0.03);
-}
+// .nav-text {
+//   font-family: $poppins;
+//   font-weight: 300;
+//   font-size: 17px;
+//   line-height: 25px;
+//   text-shadow: 0px 2px 2px rgba(0, 0, 0, 0.03);
+// }
 
-.button-text {
-  font-family: $poppins;
-  font-weight: 600;
-  font-size: 15px;
-  line-height: 22px;
-  letter-spacing: 0.03em;
-  text-transform: uppercase;
-}
+// .button-text {
+//   font-family: $poppins;
+//   font-weight: 600;
+//   font-size: 15px;
+//   line-height: 22px;
+//   letter-spacing: 0.03em;
+//   text-transform: uppercase;
+// }
```

Comment:
Why comment it out?

Reply:
Get rid of linter errors. I'll just erase it if no one is using it.

Reply:
Can you delete it? Commenting out files is no bueno